podcasts/MainTabBarController.swift

```diff
@@ -8,13 +8,13 @@ import PocketCastsUtils
 class MainTabBarController: UITabBarController, NavigationProtocol {
     enum Tab { case podcasts, filter, discover, profile, upNext }
 
-    var tabs = [Tab]()
+    var pcTabs = [Tab]()
```

Comment:
I had to do this because of:

<img width="1128" alt="Screenshot 2024-06-18 at 12 16 40" src="https://github.com/Automattic/pocket-casts-ios/assets/7040243/f391aef1-7467-4ac7-b082-c61b59086411">